src/components/behandlerdialog/meldinger/SamtaleAccordionItem.tsx

```diff
@@ -18,6 +18,7 @@ interface BehandlerNavnProps {
 
 const BehandlerNavn = ({ behandlerRef }: BehandlerNavnProps) => {
   const navn = useBehandlerNavn(behandlerRef);
+  // eslint-disable-next-line react/jsx-no-useless-fragment
   return <>{navn}</>;
```

Comment:
Denne her trenger jeg hjelp med 🙃 Test feiler når `<div>{navn}</div>`

Reply:
Kanskje det bare kan flyttes inn i hovedkompomonenten? 🤔 Eller feiler det da også? 😅

Reply:
Får ikke lov å conditionally bruke en hook, `behandlerRef` må ha en verdi for å kunne bruke hook'en😅